scratch/test_tracking.py

```
@@ -0,0 +1,92 @@
+# Let's generate a signature for click
+dest_url = "https://example.com/dest"
+u_param = base64.urlsafe_b64encode(dest_url.encode()).decode().rstrip("=")
+
+# HMAC signature base: {dispatch_id}:{u_param}
+base = f"{dispatch_id}:{u_param}"
+sig = hmac.new(tracking_secret.encode(), base.encode(), hashlib.sha256).hexdigest()
+
+# Call /track/click with ismap style coordinates: ?150,230
+click_url = f"{api_base}/track/click?d={dispatch_id}&u={u_param}&s={sig}&150,230"
+print(f"Calling click URL: {click_url}")
+response = requests.get(click_url, allow_redirects=False)
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)